ntbs-service/Pages/Shared/_Footer.cshtml

```diff
@@ -5,18 +5,18 @@
         <h2 class="govuk-visually-hidden">Support links</h2>
         <ul class="govuk-footer__inline-list footer-link-list">
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link footer-link" href="https://www.england.nhs.uk/tuberculosis-strategy-for-england-2015-2020/">
+            <a class="govuk-footer__link footer-link" href="https://www.england.nhs.uk/tuberculosis-strategy-for-england-2015-2020/" target="_blank">

```

Comment:
Close, but this is actually insecure! (Only if the nhs website is being malicious (/hijacked), but still).
https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/
`rel="noopener noreferrer"` is a must on `target="_blank"` links!
As it's not a well-advertised vulnerability: @ChristianMcc  @lbbethke @ffarmanov

Reply:
Oh damn, nice spot - thanks for changing :)